eslint-config-angular/index.mjs

```diff
@@ -48,6 +48,10 @@ export const configBase = typescriptEslint.config({
     'space-infix-ops': ['off'],
     'space-before-blocks': ['off'],
 
+    // Disable the recommended formatting rules.
+    'no-unexpected-multiline': ['off'],
+    'curly': ['off'],
```

Comment:
this should actually be enabled / as is.
I think we accidentally disabled this before by  using eslint-config-prettier.

If we use `all` (default) it does not conflict: https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#curly

Reply:
Tested it and it only requires formatting once (no feedback loop), so seems fine, removed the exception.